Update data-collection-transformations.md (#106626)
Azure/azure-content: articles/azure-monitor/essentials/data-collection-transformations.md

| Transformation in workspace DCR | [Add workspace transformation to Azure Monitor Logs by using the Azure portal](../logs/tutorial-workspace-transformations-portal.md)<br>[Add workspace transformation to Azure Monitor Logs by using Resource Manager templates](../logs/tutorial-workspace-transformations-api.md)

## Cost for transformations

> While transformations themselves do not incur direct costs, but following scenarios can result in additional charges:

Reviewer: ```suggestion
While transformations themselves don't incur direct costs, the following scenarios can result in additional charges:
```

- If a transformation increases the size of the incoming data, such as by adding a calculated column, you will be charged the standard ingestion rate for the extra data.

> - If a transformation reduces the incoming data by more than 50%, you will be charged for the amount of filtered data above 50%.

Reviewer: ```suggestion
- If a transformation increases the size of the incoming data, such as by adding a calculated column, you'll be charged the standard ingestion rate for the extra data.
- If a transformation reduces the incoming data by more than 50%, you'll be charged for the amount of filtered data above 50%.
```

> To calculate the data processing charge resulting from transformations, use the following formula: [GB filtered out by transformations] - ([Total GB ingested] / 2). For example, if you ingest 100 GB of data and your transformations remove 70 GB, you will be charged for 70 GB - (100 GB / 2), which is 20 GB. This calculation is done per data collection rule and per day basis. To avoid this charge, it is recommended to filter incoming data using alternative methods before applying transformations. By doing so, you can reduce the amount of data processed by transformations, and therefore, minimize any additional costs.

Reviewer: ```suggestion
To calculate the data processing charge resulting from transformations, use the following formula: [GB filtered out by transformations] - ([Total GB ingested] / 2). For example, if you ingest 100 GB of data and your transformations remove 70 GB, you'll be charged for 70 GB - (100 GB / 2), which is 20 GB. This calculation is done per data collection rule and per day basis. To avoid this charge, it's recommended to filter incoming data using alternative methods before applying transformations. By doing so, you can reduce the amount of data processed by transformations and, therefore, minimize any additional costs.
```
